specification/eventgrid/Azure.Messaging.EventGrid.SystemEvents/main.tsp

```diff
@@ -37,5 +37,6 @@ namespace Microsoft.EventGrid.SystemEvents {
 
   enum ServiceApiVersions {
     v2024_01_01: "2024-01-01",
+    v2018_01_01: "2018-01-01",
```

Comment:
lgtm thanks! can we also generate this -> tsp compile main.tsp

By adding this 2018 version, this will generate a systemEvents.json under 2018-01-01 as well as 2024-01-01 (we already have a file called that under 2018 from swagger) so we will need to do a rename somewhere I think -> maybe call the older SystemEvents.json -> swaggerSystemEvents.json - or rename in the tspconfig the typespec name?

Reply:
Changed to GeneratedSystemEvents